account/recovery.go

```diff
@@ -123,17 +128,254 @@ func recoverInitalState(ctx context.Context, cfg RecoveryConfig) (
 	return accounts, nil
 }
 
+// recreatePossibleAccounts returns a set of potentially valid accounts
+// by generating a set of recovery keys.
+func recreatePossibleAccounts(ctx context.Context, cfg RecoveryConfig) (
+	map[*Account]struct{}, error) {
```

Comment:
I don't think we should use pointers as map keys. Why not just use a slice here?

Reply:
Or use the account index as key if we need to be able to delete things in the map later.